lib/utils/manager/upload_manager/upload_manager.dart

```diff
@@ -105,35 +135,39 @@ class UploadManager {
       entities: entities,
     );
 
-    for (final txid in txids.keys) {
-      final fakeSendingFileInfo = txids[txid];
+    for (final txid in txids.keys.toList().asMap().entries) {
```

Comment:
why not just use `txids.entries`

Reply:
`txids.keys.toList().asMap().entries` -> `txids.entries`

Reply:
Fixed